koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -147,14 +148,14 @@ def _generate_create_table_sql(self, partition_map=None):
                 f") WITH(external_location = 's3a://{s3_path}', format = 'PARQUET',"
                 " partitioned_by=ARRAY['source', 'year', 'month'])"
             )
-        LOG.info(f"Create Parquet Table SQL: {sql}")
+        LOG.info(log_json("Create Parquet Table SQL", table=self._table_name, schema=self._schema_name, sql=sql))
         return sql
 
     def create_table(self, partition_map=None):
         """Create Trino SQL table."""
         sql = self._generate_create_table_sql(partition_map=partition_map)
         self._execute_sql(sql, self._schema_name)
-        LOG.info(f"Trino Table: {self._table_name} created.")
+        LOG.info(log_json("Trino Table created", self._table_name, self._schema_name))
```

Comment:
```suggestion
        LOG.info(log_json(msg="trino table created", table=self._table_name, schema=self._schema_name))
```
This will error since it needs kwargs.

Reply:
Thanks for the early feedback! I left the PR in draft cause I knew I needed to get give it an another look/test it. However, you highlighted a lot of the stuff I would of caught this AM.